tools/lambda-promtail/go.mod

```diff
@@ -1,6 +1,8 @@
 module main
 
-go 1.19
+go 1.22.0
```

Comment:
we should update to a newer version since there's been some security fixes, lets go with `go1.22.5`